pkg/endpoint/endpoint_test.go

```diff
@@ -727,40 +726,3 @@ type ipReleaserDummy struct{}
 func (i *ipReleaserDummy) ReleaseIP(ip net.IP) error {
 	return nil
 }
```

Comment:
Not directly related, but I think `type ipReleaserDummy` looks unused as well and could be removed.